### PR TITLE
Build airgap bundle during release

### LINKED ISSUE #785: Build default bundle for the airgap install
**Is your feature request related to a problem? Please describe.**
With #755 is landing to the main branch we'd have a way to install k0s cluster at the airgaped environment

To perform such installation end-user need to have OCI bundle of images. There is guide available in the documentation on creating such bundle, but would be beneficial to the UX to have such bundle available on the release page.

**Describe the solution you'd like**
One way is to modify release workflow to export OCI images from the cluster we setup during the e2e tests in the release workflow.

**Describe alternatives you've considered**
We can also set up completely separate workflow which would build bundle by using `docker images save` command

### PR DESCRIPTION
**Pull request by soider**
**Issue**
Fixes #785

**What this PR Includes**
GHA release.yaml changes to build bundle for airgap install during the release process for all supported architectures.

Also, includes PR #831

### CODE REVIEW

**Review by jasmingacic — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -37,6 +37,7 @@ require (
 	github.com/onsi/gomega v1.10.2 // indirect
 	github.com/opencontainers/selinux v1.8.0 // indirect
 	github.com/pkg/errors v0.9.1
+	github.com/pkg/sftp v1.13.0 // indirect
```
Comment (jnummelin) [inline go.mod:40]: I don't think the smokes are using ftp anymore?

File: inttest/footloose-alpine/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.13
 
-RUN apk add openrc openssh-server bash busybox-initscripts coreutils findutils iptables curl haproxy
+RUN apk add openrc openssh-server bash busybox-initscripts coreutils findutils iptables curl haproxy openssh-client
```
Comment (jnummelin) [inline inttest/footloose-alpine/Dockerfile:3]: Do we really need the ssh-client packages?

Comment (soider) [reply inttest/footloose-alpine/Dockerfile:3]: not anymore, I used it before I add volumes

File: .github/workflows/go.yml [hunk 3]
```diff
@@ -455,6 +455,59 @@ jobs:
...
+
+    steps:
+      - name: Get PR Reference and Set Cache Name
+        run: |
+          PR_NUMBER=$(echo ${GITHUB_REF} | cut -d / -f 3 )
+          echo "cachePrefix=k0s-${PR_NUMBER}-${{ github.sha }}" >> $GITHUB_ENV
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Bindata
```
Comment (jnummelin) [inline .github/workflows/go.yml:471]: Why do we need the embedded-bins cache for smoke? We should already have the full k0s bin pre-build and restored from cache, right?

**Review by jnummelin — APPROVED** [review]
One minor comment, otherwise LGTM

**Review by jnummelin — APPROVED** [review]

**Review by jnummelin — APPROVED** [review]

**Pull request merged**